packages/account/src/Components/account-limits/account-limits-article.jsx

```diff
@@ -1,22 +1,32 @@
+import PropTypes from 'prop-types';
 import * as React from 'react';
 import { StaticUrl } from '@deriv/components';
 import { Localize, localize } from '@deriv/translations';
 import AccountArticle from 'Components/article/article.jsx';
 
-const AccountLimitsArticle = () => {
-    return (
-        <AccountArticle
-            title={localize('Account limits')}
-            descriptions={[
-                <Localize key={0} i18n_default_text='These are default limits that we apply to your accounts.' />,
+const AccountLimitsArticle = ({ is_navigated_from_deriv_go }) => {
+    const getDescription = () => {
+        const description = [];
+        description.push(
+            <Localize key={0} i18n_default_text='These are default limits that we apply to your accounts.' />
+        );
```

Comment:
```suggestion
        const description = [
            <Localize key={0} i18n_default_text='These are default limits that we apply to your accounts.' />
        ];
```
Since this is pushed in all cases, maybe we can declare it with the `Localize`